darkside-test-lib/src/androidTest/java/cash/z/ecc/android/sdk/darkside/test/DarksideTestCoordinator.kt

```diff
@@ -144,8 +144,8 @@ class DarksideTestCoordinator(val wallet: TestWallet) {
 
         fun validateHasBlock(height: BlockHeight) {
             runBlocking {
-                assertTrue((synchronizer as SdkSynchronizer).findBlockHashAsHex(height) != null)
-                assertTrue((synchronizer as SdkSynchronizer).findBlockHash(height)?.size ?: 0 > 0)
+                assertTrue((synchronizer).findBlockHashAsHex(height) != null)
```

Comment:
Non-blocking nit:
```suggestion
                assertTrue(synchronizer.findBlockHashAsHex(height) != null)
```
and similarly in other cases.